src/Debugging/Microsoft.DiaSymReader.PortablePdb/PortablePdbReader.cs

```diff
@@ -52,7 +103,7 @@ public void Dispose()
             if (!IsDisposed)
             {
                 _pinnedImage.Free();
-                MetadataImport = null;
```

Comment:
The CLR's implementation of IMetaDataImport will leave the file locked until it is final released, which means that we unfortunately cannot just rely on the GC to collect this. So, on Windows at least, we need to do --
if (_metaDataReader.HasValue && Marshal.IsComObject(_metaDataReader.Value))
{
   Marshal.ReleaseComObject(_metaDataReader.Value);
}